npm/react/src/mount.ts

```diff
@@ -25,6 +25,21 @@ const cleanup = () => {
   return false
 }
 
+/**
+ * Mounts a React component into the DOM.
+ * @param jsx {React.ReactNode} The React component to mount.
+ * @param options {MountOptions} [options={}] options to pass to the mount function.
+ * @param rerenderKey {string} [rerenderKey] A key to use to force a rerender.
+ * @see {@link https://docs.cypress.io/guides/component-testing/mounting-react} for more details.
+ * @example
+ * import { mount } from '@cypress/react'
+ * import { Stepper } from './Stepper'
+ * it('mounts', () => {
+ *  mount(<StepperComponent />)
+ *  cy.get('[data-cy=increment]').click()
+ *  cy.get('[data-cy=counter]').should('have.text', '1')
```

Comment:
```suggestion
 * 
 * it('mounts', () => {
 *   mount(<StepperComponent />)
 *   cy.get('[data-cy=increment]').click()
 *   cy.get('[data-cy=counter]').should('have.text', '1')
```

Indentation and new line